src/Hosting/samples/SampleForHang/Program.cs

```diff
@@ -0,0 +1,22 @@
+
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+
+namespace SampleForHang
```

Comment:
Adding a new project for this wasn't necessary, remove it. A project can have multiple Program classes, there's an attribute in the project for which one to run.
https://github.com/aspnet/Extensions/blob/ab1b0aa16453b7c5876365f5dc6d476153873af0/src/Hosting/samples/GenericHostSample/ProgramFullControl.cs#L10

Reply:
Ah, I forgot about that, thanks!